policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml

```diff
@@ -14,3 +14,5 @@ metadata:
     href: https://houpl.org/wp-content/uploads/2022/01/2021_IT-511_Individual_Income_Tax_Booklet-12.29.21-Web-Version.pdf#page=13
   - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7-26 (b)(3)
     href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=5c65c45e-44b1-4c09-be93-e1cf5dc8590c&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A6348-G0H1-DYB7-W3JT-00008-00&pdcontentcomponentid=234186&pdteaserkey=sr0&pditab=allpods&ecomp=7s65kkk&earg=sr0&prid=f512a2f2-3c2b-498a-8f4c-dd1c2ee7934f
```

Comment:
document not available could you update? if the legal code includes these changes we can drop the legislation reference